src/cli/docker.rs

```diff
@@ -45,6 +45,36 @@ impl Docker {
         Ok(())
     }
 
+    // Build & run docker image
+    pub fn build_run() -> Result<(), Box<dyn Error>> {
+        let mut command = String::from("docker build . -t postgres");
```

Comment:
Any reason to not use this?

```suggestion
        let command = "docker build . -t postgres && docker run -p 5432:5432 -d postgres";
        match run_command(command) {
```

Reply:
Just better readability but I had a third command here as well that I refactored into another function so will change it as per your suggestion.